pkg/util/gitlab/repository_files.go

```diff
@@ -13,7 +15,13 @@ func (c *Client) FileExists(project, branch, filename string) (bool, error) {
 	}
 
 	_, response, err := c.RepositoryFiles.GetFile(project, filename, getFileOptions)
-	if response.StatusCode == 404 {
+	if response.StatusCode == http.StatusBadRequest {
```

Comment:
Perhaps a loop can be used to solve more situations in the future

Reply:
@jxs1211 If you get stuck or don't have the time, I can provide you with a rough copy of the code.

```go
for _, code := range []int{http.StatusBadRequest, http.StatusUnauthorized, http.StatusNotFound} {
	if response.StatusCode == code {
		return false, nil
	}
}
```

Reply:
@iyear obviously, this way is better than the previous one, let me update it.